### DISCUSSION TITLE
Images inside repository stopped loading

### DISCUSSION THREAD

**Discussion by AnkushSinghGandhi** [community/community#42410] [Repositories]
### Select Topic Area

Bug

### Body

Everythin was working great but few days back I noticed my profile readme stoped showing banner image which was prsesnt in the same profile repo. and this is happening in all of my github repositories the images in readme stopped loading.

all of my repositories suddeny stopped loading images

![Screenshot from 2022-12-22 22-25-42](https://user-images.githubusercontent.com/55637484/209187340-e931820c-7c8f-44c8-b5ad-933b396ca922.png)

![Screenshot from 2022-12-22 22-27-05](https://user-images.githubusercontent.com/55637484/209187366-fe9833ab-30c8-46b7-a18b-1d1d67dd554a.png)

![Screenshot from 2022-12-22 22-27-33](https://user-images.githubusercontent.com/55637484/209187389-eea140eb-9a35-4082-adac-dd61f74f3438.png)

[profile repo](https://github.com/AnkushSinghGandhi/AnkushSinghGandhi)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @AnkushSinghGandhi ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
